app/chatpal-search/server/utils/utils.js

```diff
@@ -1,10 +1,21 @@
 import { Meteor } from 'meteor/meteor';
 import { HTTP } from 'meteor/http';
 
+import ChatpalLogger from './logger';
+import { settings } from '../../../settings';
+
 Meteor.methods({
 	'chatpalUtilsCreateKey'(email) {
 		try {
-			const response = HTTP.call('POST', 'https://beta.chatpal.io/v1/account', { data: { email, tier: 'free' } });
+			const data = {
+				email,
+				instanceId: settings.get('uniqueID'),
```

Comment:
Curious the usage for this?  If counting on it to be unique... better off generating your own.  UniqueId generated is unique but sometimes people copy their dbs to spin up new workspaces (I’m guessing to simplify setup)

Reply:
@geekgonecrazy the uniqueID is being used in order to identify the tenant / workspace / server the data belongs to.
If Somebody copies the database, they will also copy the rest of the data. So I'd consider this appropriate in this case.